tests/Unit/BranchAliasTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Test\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Verify BranchAlias values.
+ * Rules:
+ * - For component without stable release (ie. 0.2.3):
+ *   - When the ChangeLog file contains a major change => 0.3-dev.
+ *   - When the ChangeLog file contains a minor change => alias absent.
+ *   - When the ChangeLog file contains a patch change or no change => alias absent.
+ * - For component with stable release (ie. 1.2.3):
+ *   - When the ChangeLog file contains a major change => 2.0-dev.
+ *   - When the ChangeLog file contains a minor change => 1.3-dev.
+ *   - When the ChangeLog file contains a patch change or no change => alias absent.
+ */
```

Comment:
is it correct?

Reply:
No. `When the ChangeLog file contains a patch change or no change => alias absent.` should be that the alias matches the major.minor from the last release.